analysis/tests/src/Rename.res

```diff
@@ -9,3 +9,6 @@ let c = x
 
 let foo = (~xx) => xx + 1
 //                 ^ren yy
+
+let foo2 = (~xx) => xx + 1
+//            ^ren yy
```

Comment:
So `foo` was working and `foo2` wasn't?
Not sure I get what's going on.

Reply:
They both work, but when for `foo` the tilde is dropped from `~xx` so it becomes `(yy) => ...`, and for `foo2` the tilde was added to the body `(~yy) => ~yy + 1`.

Reply:
Perhaps hacking renaming itself to special-case tilde does not break other things.

Reply:
That was my first idea but it might be a little bit of extra work, because we'll need to read the text from the loc and have a look at that. There's no information carried on that can help us when producing the rename instructions. Might be the best idea anyway though.

Reply:
> That was my first idea but it might be a little bit of extra work, because we'll need to read the text from the loc and have a look at that. There's no information carried on that can help us when producing the rename instructions. Might be the best idea anyway though.

Not sure it makes sense, but perhaps when the length does not match. E.g. if the loc says 3 and it's supposed to be 2. Though that's risky.